<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>803</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2979FF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_titulo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Tabla de Multiplicar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_numTabla">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Tabla del :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Unidad2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Unidad 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenGatos">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>600</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Images/Gatos.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_numTabla_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Tu Tabla es:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Tabla">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>311</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenUat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/log_uat_nuevo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Valor">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
border-radius:25px;
border:8px solid #B0BEC5;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32774</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenFi">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/FIT_logo_vertical.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CT">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>Crear Tabla</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Archivos/Recursos.qrc"/>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
